### PR TITLE
V2.0.0

### PR DESCRIPTION
**Pull request by ocefpaf**
This will probably fail and we'll need to work on it a little bit to smooth out the corners.

Ping @pelson

### CODE REVIEW

**Comment by ocefpaf** [comment]
@pelson I guess that the tarball is missing something, no?

**Comment by pelson** [comment]
Change looks good. Something is well borked though. I'll double check what's going on...

**Comment by ocefpaf** [comment]
> Change looks good. Something is well borked though. I'll double check what's going on...

The PyPI tarball is mising the `.pyd` file. I changed it here to use the GitHub tarball now and sent a PR to cf_units that hopefully fixes this. See https://github.com/SciTools/cf_units/pull/104

**Comment by pelson** [comment]
Confirming I get the same thing locally 😞

**Comment by pelson** [comment]
Let's go for cf_units v2.0.1 😉 
https://pypi.org/project/cf_units/

**Comment by ocefpaf** [comment]
Windows is still failing but I have a hunch that is due `udunits2` fault and not cf_units. See https://ci.appveyor.com/project/conda-forge/cf-units-feedstock/build/1.0.31/job/9efx4k73a0plladl

File: recipe/build.sh [hunk 1]
```diff
@@ -1,9 +1,7 @@
 #!/bin/bash
 
-# Make sure cf_units can find the udunits library.
 SITECFG=cf_units/etc/site.cfg
```
Comment (pelson) [inline recipe/build.sh]: I recon we can remove the site.cfg file for Linux & OSX.

**Comment by pelson** [comment]
Setting the environment variable UDUNITS2_XML_PATH should have an impact on whether the tests pass. This wouldn't normally be necessary, but there is a bug with the UDUINTS build, as described at https://github.com/conda-forge/udunits2-feedstock/issues/18.

Don't think we need to hold up this PR to fix that issue, if we can get the tests passing using the UDUNITS2_XML_PATH env var (with a comment referencing the issue, so that we know when we can remove the restriction).

File: recipe/build.sh (hunk 1 above)
Comment (ocefpaf) [reply recipe/build.sh]: I left it there for "symmetry" no strong opinions and glad to remove it.

**Comment by ocefpaf** [comment]
> Don't think we need to hold up this PR to fix that issue, if we can get the tests passing using the UDUNITS2_XML_PATH

~~Testing that now. If the env var works I'll fix `udunits2` feedstock with an activate and deactivate script like the one we have for `*nix`.~~

---
Edit: I never committed the `scripts` solution :flushed: It was rotting in a branch.
Just sent https://github.com/conda-forge/udunits2-feedstock/pull/19 to implement that for `*nix` and `Windows`.

**Comment by ocefpaf** [comment]
Working on a version of `udunits2 2.2.25 vc9` with the same scripts now, so we can have cf_units on Python 2.7+Win too.

File: recipe/bld.bat [hunk 2]
```diff
@@ -1,7 +0,0 @@
-set SITECFG=cf_units/etc/site.cfg
-echo [System] > %SITECFG%
-echo udunits2_path = %SCRIPTS%\udunits2.dll >> %SITECFG%
-echo udunits2_xml_path = %LIBRARY_PREFIX%\share\udunits\udunits2.xml >> %SITECFG%
```
Comment (pelson) [inline recipe/bld.bat:4]: Personally, I'd have kept this, as it helps cf_units work even if users don't activate their environment.
That said, I'm not overly concerned about it 😄

**Comment by pelson** [comment]
Awesome stuff @ocefpaf!!

File: recipe/bld.bat (hunk 2 above)
Comment (ocefpaf) [reply recipe/bld.bat:4]: Makes sense. I can re-add that in another PR later. I guess I was confused due to your comment on this line for Linux and OS X before. Should I re-add `build.sh` and `bld.bat` with the `cf_units/etc/site.cfg` and the `udunits2.xml` path on both?

Comment (pelson) [reply recipe/bld.bat:4]: Sorry, confusing messages. conda does the binary replacement just fine for OSX and Linux, just not for Windows. Happy to have the config file for all platforms though for consistency.

Comment (ocefpaf) [reply recipe/bld.bat:4]: No problem.

**Pull request merged**